<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lottery</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="blueNumber">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>88</number>
    </property>
   </widget>
   <widget class="QPushButton" name="hopeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>机选</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>彩票选号器</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLCDNumber" name="redNumber_1">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="redNumber_2">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="redNumber_3">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="redNumber_4">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="redNumber_5">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="redNumber_6">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="fliterBox_6">
       <property name="text">
        <string>过滤器1</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_7">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_8">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_9">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_10">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="fliterBox_1">
       <property name="text">
        <string>过滤器1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_2">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_3">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_4">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fliterBox_5">
       <property name="text">
        <string>过滤器1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
